### ISSUE TITLE
gbridge: tcpip + tls: concentrate traffic into a single tcp/ip connection

### ISSUE THREAD

**Issue by cfriedt** [cfriedt/gbridge#8]
This ticket is intended to mirror cfriedt/greybus-for-zephyr#55.

TL;DR - using 1 TCP/IP port per Greybus CPort forces an excessive drain on memory resources on a microcontroller with only kB of RAM. This is particularly true for TLS when mutual authentication is used, as client keys must be stored for each TLS connection.

Like the UART transport, the TCP/IP and TLS transports should probably stuff the CPort ID inside of one of the reserved fields inside of the Greybus Message Header.

This feature can be viewed as an optimization (a space optimization in terms of reducing dynamic memory).

For details, see [cport_pack() / cport_unpack()](https://github.com/cfriedt/gbridge/blob/master/controller.c#L36).
